libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -26,10 +26,13 @@
 #include "dnn_backend_openvino.h"
 #include "libavformat/avio.h"
 #include "libavutil/avassert.h"
+#include "libavutil/opt.h"
+#include "../internal.h"
 #include <c_api/ie_c_api.h>
 
 typedef struct OVContext {
     const AVClass *class;
+    char *device_type;
```

Comment:
there might be more options, so let's add code like:
struct OVOptions
{
char *device_type;
}

Reply:
um, as I know the options are setted in struct XXXContext generally, so the funcs (in dict.c and opt.c) can assign values to membors of XXXContext and show error log info with ctx.
If we create new struct like OVOption, and pass it to funcs(in dict.c & opt.c), the error log info will not be shown correctly.

Reply:
i guess it is possible, to pass OVContext.options.device_type.  The offset is OFFSET(options.device_type) instead of OFFSET(device_type)